.github/workflows/CI.yml

```diff
@@ -52,56 +53,49 @@ jobs:
         rust-toolchains:
           - 1.65.0
           - stable
+          - beta
```

Comment:
I think I removed `beta` from `master` CI on purpose; `future-proof` takes care of covering it (since it's more of a "future `stable`" right?

```suggestion
```

Reply:
Sounds good!